lib/std/Build/Module.zig

```diff
@@ -171,6 +172,9 @@ pub const IncludeDir = union(enum) {
                 const header_dir_path = full_file_path[0 .. full_file_path.len - config_header.include_path.len];
                 try zig_args.appendSlice(&.{ "-I", header_dir_path });
             },
+            .embed_path => |embed_path| {
+                try zig_args.append(try std.mem.concat(b.allocator, u8, &.{ "--embed-dir=", embed_path.getPath2(b, asking_step) }));
```

Comment:
```suggestion
                try zig_args.appendSlice(&.{ "--embed-dir=", embed_path.getPath2(b, asking_step) });
```
see how the one above this does it.

Reply:
--embed-dir=<DIR> is one argument, not two; using appendSlice will add two different arguments, causing an error

Reply:
Ah, you're right, I missed that `=`.